src/metabase/task/truncate_audit_log.clj

```diff
@@ -0,0 +1,75 @@
+(ns metabase.task.truncate-audit-log
```

Comment:
Kept the namespace generic instead of something like `truncate-query-execution` since we eventually want to use this mechanism for other audit data as well